src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenAsyncTests.cs

```diff
@@ -27,9 +27,13 @@ public class CodeGenAsyncTests : EmitMetadataTestBase
         private const MethodImplAttributes MethodImplOptionsAsync = (MethodImplAttributes)1024;
         private static CSharpParseOptions WithRuntimeAsync(CSharpParseOptions options) => options.WithFeature("runtime-async", "on");
 
-        internal static string ExpectedOutput(string output)
+        internal static string ExpectedOutput(string output, bool isRuntimeAsync = false)
         {
-            return ExecutionConditionUtil.IsMonoOrCoreClr ? output : null;
+            return ExecutionConditionUtil.IsMonoOrCoreClr
+                ? isRuntimeAsync
+                    ? null
```

Comment:
Consider logging an issue to enable execution once we build with a runtime that supports runtime async.

Reply:
I'll leave a prototype comment for now, I don't know which part will come first, branch merge or runtime with support.